lib/cli/node-modules-path.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var path  = require('path');
+
+module.exports = function(context) {
+  var nodePath = process.env.EMBER_NODE_PATH;
```

Comment:
should we just try to observer `NODE_PATH` directly then? I imagine this would align ember-cli with node's own behavior more closely? Or is this going to far?